composer.json

```diff
@@ -97,8 +96,15 @@
         "beberlei/DoctrineExtensions": "^1.0",
         "composer/ca-bundle": "^1.0",
         "nikic/php-parser": "^2.1",
-        "prestashop/decimal": "^1.0.0"
+        "prestashop/decimal": "^1.0.0",
+        "cssjanus/cssjanus": "dev-patch-1"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/prestashop/php-cssjanus"
```

Comment:
Can you please describe what is the reason of that fork?

Reply:
Long story short -- it doesn't handle some CSS files. You can learn more here: https://github.com/cssjanus/php-cssjanus/pull/13

The idea is to come back to the original library if/when the maintainer decides to merge that PR.